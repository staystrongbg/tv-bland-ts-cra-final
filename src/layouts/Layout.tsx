import Footer from '../components/Footer/Footer';
import Navbar from '../components/Navbar/Navbar';
import { Outlet } from 'react-router-dom';

const Layout: React.FC = () => {
  return (
    <>
      <Navbar />
      <Outlet />
      <Footer />
    </>
  );
};

export default Layout;
