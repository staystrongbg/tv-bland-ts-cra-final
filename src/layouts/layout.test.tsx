import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Layout from './Layout';

it('render app', () => {
  const render = renderer
    .create(
      <BrowserRouter>
        <Layout />
      </BrowserRouter>
    )
    .toJSON();
  expect(render).toMatchSnapshot();
});
