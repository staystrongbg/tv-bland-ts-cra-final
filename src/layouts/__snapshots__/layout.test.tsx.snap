// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
Array [
  <nav
    className="flex-between"
    style={
      Object {
        "backgroundColor": "#101010",
        "color": "#f1f2f3",
        "padding": "20px 60px",
      }
    }
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1>
        TV Bland
      </h1>
    </a>
  </nav>,
  <footer
    className="footer"
  >
    <p>
      TV Bland
    </p>
    <p>
      Copyright TV BlandÂ©2022
    </p>
  </footer>,
]
`;
